students/hirot_directory/session03/mailroom.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+donor_names = ["William Gates III", "Mark Zuckerberg", "Jeff Bezos", "Paul Allen"]
+donations = [[200,500], [2000,3000], [4000,5000], [100,150]]
+
+
+def thank_you():
+
+	while True:
+
+		if (str(input("Do you want to see the donor list? if yes, type List: ")) == "list"):
+			print(donor_names, donations)
+		new_donor = str(input("Full Name: "))
+		new_donation = int(input("How much do you want to donate?: "))
+	
+		if (new_donor not in donor_names):
+			donor_names.append(new_donor)
+			donations.append([new_donation])
+			print("Thank you for donation, {}! Your donation amount is {}.".format(new_donor, new_donation))
+
+		else:
+			number = donor_names.index(new_donor) #4
+			donations[number].append(new_donation)
+			print("Thank you donation, {}! Your donation amount is {}.".format(new_donor, new_donation))
```

Comment:
Yes, I noticed it too while I was writing but I got too confused on this assignment so I decided to keep it as it is since I was scared to break codes that were working... But I will do next time.